test/integration/targets/cloudwatchlogs/tasks/main.yml

```
@@ -0,0 +1,138 @@
+---
+
+- module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
```

```suggestion
      security_token: "{{ security_token | default(omit) }}"
```

Security Tokens are only used in some contexts.  It's less likely to cause issues for reviewers if you make security_token optional.